.github/actions/verify-token/src/predicate.ts

```diff
@@ -118,10 +113,21 @@ export interface githubObj {
   actor: string;
 }
 
+function getWorkflowPath(repoName: string, workflowRef: string): string {
+  // GITHUB_WORKFlOW_REF contains the repository name in the path. We will trim
+  // it out.
+  let ref = workflowRef.trim();
+  const repoPrefix = `${repoName.trim()}/`;
+  if (ref.startsWith(repoPrefix)) {
```

Comment:
is it always the case that ref starts with the repo name? Is this something we should validate?

Reply:
I added a check here so it fails if the value isn't what is expected. I plan to update the validation logic so I will send a follow up PR that will pre-validate this.